docs/new-curriculum/js-core-1/week-1/notes.md

```diff
@@ -0,0 +1,349 @@
+The actual combination of symbols `10 + 32` we say is an **expression**.
+
+:::info
+An **expression** is a value or any valid combination of values and symbols that results in a single value.  
+We say that expressions **evaluate** to a single value.  
+So we say that `10 + 32` **evaluates** to the value `42`.
+:::
+
+### Evaluating expressions
+
+So we can look at an expression like `36 * 45` and ask: what does this expression evaluate to? If we understand what the `*` operator represents ( in this case multiplication ) and if we understand some basic rules of arithmetic we can evaluate this expression.  
+However, we can use computers to evaluate expressions. We can use NodeJS, a JavaScript runtime, an application that runs JavaScript programs.
+
+Node can be used in a number of different ways; however, one particular mode is the REPL. The Node REPL enables us to input JavaScript expressions which are then evaluated by NodeJS and printed to the terminal. The REPL will print out the result of this evaluation.
+
+:::note
+
+ℹ️ Definition: REPL is a special type of program that stands for
+
+**Read**  
+**Evaluate**  
+**Print**  
+**Loop**
+
+:::
+
+We can type each of the following expressions into the REPL and then press enter to check the result.
+
+```bash
+10 + 32
+```
+
+```bash
+32 / 10
+```
+
+```bash
+3 ** 4
+```
+
+### Saving expressions
+
+Often in programming we may want to reuse a particular expression in our program. In this case, we need a way to refer back to a particular expression. Let’s consider the following string:
+
+```js
+"Hello there";
+```
+
+Suppose we want to reuse this string in several different places. For example, we may want to create different greetings for different users, like:
+
+```bash
+"Hello there, Alicia";
+```
+
+```bash
+"Hello there, Barny"
+```
+
+We can use a **variable** to store this string value so it can be referenced and used again.
+
+:::info
+
+A **variable** is a label for a piece of data. We can assign a piece of data to a particular label and then refer back to this label.
+
+:::
+
+We can create a variable in our program by writing a **variable declaration**, in the following way:
+
+```js title="variable declaration"
+const greeting = "Hello there";
+```
+
+We can break down the different syntactic elements of this variable declaration:
+
+- `const` is a keyword used to indicate we're creating a variable.
+- `greeting` is the variable name - like the name of the label for our piece of data.
+- `=` this is the assignment operator. It means assign to the label `greeting` the expression on the right hand side.
+- `"Hello there"` - this is the value we're assigning to the label `greeting`.
+
+We can type this variable declaration into the REPL:
+
+![greeting-variable-declaration](./assets/greeting-variable-declaration.png)
```

Comment:
This shows the variable declaration returning `undefined`. I feel pretty confident that it'll come up as a question.

I had to [double check this](https://stackoverflow.com/questions/49470717/mac-terminal-node-repl-output-for-variable-declarations), but this is because using `const` to declare the variable makes it a statement, not an expression, so it returns `undefined`.

This does not seem like it is easy to explain, so I guess we should just tell trainees to ignore it? Doesn't feel very satisfying to me.